test/units/modules/network/eos/test_eos_static_routes.py

```diff
@@ -0,0 +1,206 @@
+                                                                    admin_distance=23)]
+                                                    )]
+                                      )]
+            )], state="replaced"
+        ))
+        self.execute_module(changed=False, commands=[])
+
+    def test_eos_static_routes_overridden(self):
+        set_module_args(dict(
+            config=[dict(
+                vrf="testvrf",
+                address_families=[dict(afi="ipv6",
+                                       routes=[dict(dest="1200:10::/64",
+                                                    next_hops=[dict(interface="Ethernet1",
+                                                                    admin_distance=55)]
+                                                    )]
+                                      )]
+            )], state="overridden"
+        ))
+        commands = ['ipv6 route vrf testvrf 1200:10::/64 Ethernet1 55',
+                    'no ip route vrf testvrf 120.1.1.0/24 Ethernet1 23',
+                    'no ip route vrf vrftest1 77.77.1.0/24 33.1.1.1',
+                    'no ip route 10.1.1.0/24 Ethernet1 20.1.1.3 track bfd 200',
+                    'no ip route 10.1.1.0/24 Management1',
+                    'no ip route 10.50.0.0/16 Management1',
+                    'no ip route 23.1.0.0/16 Nexthop-Group testgrp tag 42']
+        self.execute_module(changed=True, commands=commands)
+
+    def test_eos_static_routes_overridden_idempotent(self):
+        set_module_args(dict(
+            config=[dict(
+                vrf="testvrf",
+                address_families=[dict(afi="ipv4",
+                                       routes=[dict(dest="120.1.1.0/24",
+                                                    next_hops=[dict(interface="Ethernet1",
+                                                                    admin_distance=23)]
+                                                    )]
+                                      )]
+            )], state="overridden"
+        ))
+        self.execute_module(changed=False, commands=[])
+
+    def test_eos_static_routes_deletedvrf(self):
+        set_module_args(dict(
+            config=[dict(
+                vrf="testvrf",
+            )], state="deleted"
+        ))
+        commands = ['no ip route vrf testvrf 120.1.1.0/24 Ethernet1 23']
+        self.execute_module(changed=True, commands=commands)
+
+    def test_eos_static_routes_deletedroute(self):
+        set_module_args(dict(
+            config=[dict(
+                vrf="testvrf",
+                address_families=[dict(afi="ipv4",
+                                       routes=[dict(dest="120.1.1.0/24")]
+                                      )]
+            )], state="deleted"
+        ))
+        commands = ['no ip route vrf testvrf 120.1.1.0/24 Ethernet1 23']
+
+        self.execute_module(changed=True, commands=commands)
+
+    def test_eos_static_routes_deletedafi(self):
```

Comment:
Add test cases for states rendered/merged/parsed

Reply:
Will create a PR and work on it.